src/Microsoft.OData.Edm/Schema/EdmEntityType.cs

```diff
@@ -84,6 +87,14 @@ public virtual IEnumerable<IEdmStructuralProperty> DeclaredKey
             get { return this.declaredKey; }
         }
 
+        /// <summary>
+        /// Gets the aliases for the declared key properties of the entity type.
+        /// </summary>
+        public virtual IEnumerable<KeyValuePair<IEdmStructuralProperty, string>> DeclaredKeyAlias

```

Comment:
I'm curious to know what the structural property that's returned from this field look like, what would be its `DeclaringType` be set as? In practice the property will be declared on different complex type or entity type other than the one that `this` refers to. So how do we capture that fact here? How can we tell which property of the entity contains the property that's being referenced by the alias?